### PR TITLE
Feature/correct german translation

### PR DESCRIPTION
**Pull request by XanatosX**
#### Short description of what this resolves:

Improving the translation for the german language

#### How Has This Been Tested?

I did not test the changes but did check the json file with the following validation tool: https://jsonformatter.curiousconcept.com/

#### Checklist:
- [ X] I have read the [contribution](https://github.com/shanalikhan/code-settings-sync/blob/master/CONTRIBUTING.md#setup-extension-locally) guidelines.

### CODE REVIEW

File: package.nls.de.json [hunk 1]
```diff
@@ -1,142 +1,142 @@
...
+  "cmd.downloadSettings.info.gotLatestVersion": "Sync: Du hast bereits die neueste Version der gespeicherten Einstellungen.",
+  "cmd.downloadSettings.info.readdingOnline": "Sync: Lese Online Einstellungen.",
   "cmd.downloadSettings.title": "Sync: Download-Einstellungen",
-  "cmd.howSetting.title": "Sync: Wie Konfigurieren",
-  "cmd.otherOptions.customizedSync": "Synchronisieren: Hinzufügen von Benutzerdefinierten Sync Datei-Pfad",
-  "cmd.otherOptions.customizedSync.done": "Sync: {0} registriert worden ist.",
-  "cmd.otherOptions.customizedSync.placeholder": "Geben Sie einen absoluten Pfad für die zusätzlichen hochgeladenen Datei.",
-  "cmd.otherOptions.customizedSync.prompt": "(ex. /Pfad/zu/.eslintrc) werden hoch-und runtergeladen werden in den angegebenen Pfad.",
-  "cmd.otherOptions.downloadCustomFile": "Synchronisieren: Importieren Sie Benutzerdefinierte Sync-Datei in den workspace",
+  "cmd.howSetting.title": "Sync: Wie soll es konfiguriert werden",
```
Comment (XanatosX) [inline package.nls.de.json]: I'm not sure in which content this title is getting shown so maybe the translation is not the best in this situation.

Can you tell me how to trigger this message so I can take a look?

**Comment by shanalikhan** [comment]
There are conflicts with the upcoming release branch because someone already improved it.

I would suggest you to send PR to that branch if you think it can be improved or close the PR

**Comment by XanatosX** [comment]
I think there was only one line changed:

https://github.com/shanalikhan/code-settings-sync/pull/1008/files#diff-dfd6bf5e17c76cfb02934a114c90e9f1


Or did I check the wrong PR?

**Comment by shanalikhan** [comment]
> one line changed

Yes , the that key is going to remove from all languages file.

**Comment by XanatosX** [comment]
Alright I will resolve the conflict, thank you for changing the target branch already

**Pull request merged**